src/blueprint/from_config.rs

```diff
@@ -202,6 +204,7 @@ fn to_field(
   let field_definition = update_http(field, field_definition, config).trace("@http")?;
   let field_definition = update_unsafe(field.clone(), field_definition);
   let field_definition = update_inline_field(type_of, field, field_definition, config).trace("@inline")?;
+  let field_definition = update_const_field(field, field_definition, config).trace("@const")?;
```

Comment:
Please ensure comprehensive test coverage when combining @const, @inline, and @modify decorators. No matter the order of their application, the result should remain consistent.

Add tests to confirm that when @const and @unsafe are used together, @unsafe can still leverage the constant value for data manipulation.

We need tests to demonstrate that the combination of @const and @http should fail.

Reply:
refer this
https://github.com/tailcallhq/tailcall/blob/main/tests/graphql/passed/test-inline-with-modify-field.graphql
and this for faliures
https://github.com/tailcallhq/tailcall/blob/main/tests/graphql/errors/test-inline.graphql

Reply:
Convered with tests in recent commit.